src/main/java/internet/market/service/ProductServiceImpl.java

```diff
@@ -0,0 +1,48 @@
+package internet.market.service;
+
+import internet.market.dao.ProductDao;
+import internet.market.lib.Inject;
+import internet.market.lib.Service;
+import internet.market.model.Product;
+import java.util.List;
+
+@Service
+public class ProductServiceImpl implements ProductService {
+
+    @Inject
+    private ProductDao productDao;
+
+    @Override
+    public Product create(Product product) {
+        return productDao.create(product);
+    }
+
+    @Override
+    public Product get(Long id) {
+        if (productDao.get(id).isPresent()) {
+            return productDao.get(id).get();
+        }
+        throw new IllegalArgumentException("We don't have product with such id");
```

Comment:
we can omit this check for now

let's just call `productDao.get(id).get();`

Reply:
done